Whiskey/en-US/about_Whiskey_MergeFile_Task.help.txt

```diff
@@ -0,0 +1,75 @@
+TOPIC
+    about_Whiskey_MergeFile_Task
+
+SUMMARY
+    Merges multiple files into a single, new file.
+
+DESCRIPTION
+    The `MergeFile` task takes one or more source files and merges them together into a destination file. Pass the list of source files to merge to the `Path` property (wildcards are supported). Pass the path to the destination file where the files should be merged to the `DestinationPath` property. The destination file will be created if it doesn't exist. If the destination file already exists, its contents will be preserved. To clear the contents of the destination file before merging in other files, set the `Clear` property to `true`.
+
+    By default, the source files are left in place. If you'd like to delete the source files as they are merged into the destination file, set the `DeleteSourceFiles` property to `true`.
+
+    Files are concateneated as-is. If you want to ensure each file is separated from other files appropriately, set either the `TextSeparator` property to a string to use to separate each file, or the `BinarySeparator` property to an array of bytes (i.e. numbers between 0 and 255). After merging each file, the task adds the separator. If both `TextSeparator` and `BinarySeparator` are given, `TextSeparator` takes precedence.

```

Comment:
This is no longer valid: `If both TextSeparator and BinarySeparator are given, TextSeparator takes precedence.`